src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, FigurePrinter {
+    protected String color;
```

Comment:
let's make it private and create a constructor that accepts color
then call parent constructor in child figures using `super` keyword

you can also create a public getter for color